stride/physics/boundaries/devito.py

```diff
@@ -5,6 +5,43 @@
 from .boundary import Boundary
 
 
+class SpongeBoundaryElastic(Boundary):
+
+    def __init__(self, grid):
+        '''
+        Sponge boundary for elastic codes
+        :param parameter: 0.2763 for homogeneous water, 0.008635 alpha2D
```

Comment:
this parameter comment should be removed

Reply:
Comment has been removed, not sure how best to remove the parameter. Currently I optimise the parameter by grid-search for each model

Reply:
Correction made